package.json

```
@@ -19,7 +19,7 @@
     "force-webui-download": "shx rm -rf assets/webui && run-s build:webui",
     "build": "run-s clean build:webui",
     "build:webui": "run-s build:webui:*",
-    "build:webui:download": "npx ipfs-or-gateway -c bafybeihatzsgposbr3hrngo42yckdyqcc56yean2rynnwpzxstvdlphxf4 -p assets/webui/ -t 360000 --verbose -g \"https://trustless-gateway.link\" ",
+    "build:webui:download": "npx ipfs-or-gateway -c bafybeideglc722hiwhsy4kiyl2fivf5lr6wozy2iuixtgzkvl3v4hasaty -p assets/webui/ -t 360000 --verbose -g \"https://trustless-gateway.link\" ",
```

https://github.com/ipfs/ipfs-webui/releases/tag/v4.3.2
```suggestion
    "build:webui:download": "npx ipfs-or-gateway -c bafybeif6abowqcavbkz243biyh7pde7ick5kkwwytrh7pd2hkbtuqysjxy -p assets/webui/ -t 360000 --verbose -g \"https://trustless-gateway.link\" ",
```